engine/servermaster/jobmanager.go

```diff
@@ -328,7 +371,7 @@ func buildPBJob(masterMeta *frameModel.MasterMetaKVData) (*pb.Job, error) {
 		return nil, errors.Errorf("job %s has unknown type %v", masterMeta.ID, masterMeta.Tp)
 	}
 
-	var jobStatus pb.Job_Status
+	var jobStatus pb.Job_State
```

Comment:
```suggestion
	var jobState pb.Job_State
```

Reply:
So many variable names need to be normalized.
May be we can do it in another pr.

Reply:
Please create an issue for normalizing variable names, maybe with https://github.com/pingcap/tiflow/pull/6930 together.